src/tokenizer/interface.rs

```diff
@@ -110,7 +110,8 @@ pub trait TokenSink {
 
     /// The tokenizer will call this after emitting any start tag.
     /// This allows the tree builder to change the tokenizer's state.
-    /// By default no state changes occur.
+    /// By default a <script> element will put the tokenizer into a
+    /// quiescent state, but no other changes occur.
```

Comment:
That's true. I guess I'm not sure what documentation changes you're looking for here.